Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -217,8 +224,7 @@ namespace GradientSignal
             behaviorContext->Class<ImageGradientConfig>()
                 ->Constructor()
                 ->Attribute(AZ::Script::Attributes::Category, "Vegetation")
-                ->Property("tilingX", BehaviorValueProperty(&ImageGradientConfig::m_tilingX))
-                ->Property("tilingY", BehaviorValueProperty(&ImageGradientConfig::m_tilingY))
+                ->Property("tiling", BehaviorValueProperty(&ImageGradientConfig::m_tiling))
                 ;
```

Comment:
Just verifying - are there any python tests that used these that need to be updated?

Reply:
Nope, no mention to "tiiling" in any tests under `o3de\AutomatedTesting\Gem\PythonTests` :)